safe-global/safe-homepage
# fix: Fetch ecosystem data

## What it solves

Part of #176 

- Removes `ecosystem-data.json` and all ecosystem logos inside `public/images/ecosystem`
- Fetches data and logos from staging
- Displays the description inside project card

## How to test

1. Open `/ecosystem`
2. Observe all projects and logos being there
3. Use the filter and search and observe they are working

## Screenshot

<img width="1231" alt="Screenshot 2023-05-08 at 16 28 20" src="https://user-images.githubusercontent.com/5880855/236850745-f0b21b55-f358-4cbe-8630-776293825e52.png">
